### PR TITLE
Added RHEL CoreOS tag to common attributes

### PR DESCRIPTION
**Pull request by mburke5678**
we should set this as an attribute in the common-attributes file and let the team know.

https://bugzilla.redhat.com/show_bug.cgi?id=1683600

### CODE REVIEW

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL. Vikram asked me to add an attribute to common-attributes for RHEL CoreOS. Want to make sure I am doing it right.

**Comment by kalexand-rh** [comment]
@mburke5678, can we have a short and long version? The final names are up in the air, but the last I heard, RHCOS was acceptable to use after the first mention. Maybe something like:

:op-system-first: RHEL CoreOS (RHCOS)
:op-system: RHCOS

**Comment by mburke5678** [comment]
@kalexand-rh PTAL

**Comment by kalexand-rh** [comment]
LGTM!

**Comment by mburke5678** [comment]
@vikram-redhat PTAL

**Comment by mburke5678** [comment]
@vikram-redhat

File: modules/nodes-nodes-viewing-listing.adoc [hunk 1]
```diff
@@ -114,7 +114,7 @@ System Info:    <9>
  System UUID:                             EC22BF97-A006-4A58-6AF8-0A38DEEA122A
  Boot ID:                                 f24ad37d-2594-46b4-8830-7f7555918325
  Kernel Version:                          3.10.0-957.5.1.el7.x86_64
- OS Image:                                Red Hat CoreOS 4.0
+ OS Image:                                RHEL CoreOS 4.0
```
Comment (vikram-redhat) [inline modules/nodes-nodes-viewing-listing.adoc]: @mburke5678 would you need to change this to use the attribute? I am not sure what the official OS name is?

Comment (mburke5678) [reply modules/nodes-nodes-viewing-listing.adoc]: @vikram-redhat That string is in a clode block. The attribute won't work.

**Pull request merged**